index.js

```diff
@@ -281,27 +301,33 @@ class JenkinsExecutor extends Executor {
         );
 
         const xml = this._loadJobXml(config, annotations);
+        // prettier-ignore
```

Comment:
Prettier output errors about this line.

```
  306:17  error  Insert `··`  prettier/prettier
  307:17  error  Insert `··`  prettier/prettier
  308:1   error  Insert `··`  prettier/prettier
```

But I could not resolve this errors... so, added `prettier-ignore`.